File: src/main/java/com/technews/TechNewsApplication.java
package com.technews;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@SpringBootApplication
public class TechNewsApplication {

    public static void main(String[] args) {
        SpringApplication.run(TechNewsApplication.class, args);
    }

}

File: src/main/kotlin/com/technews/TechNewsApplication.kt
package com.technews

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@EnableScheduling
@SpringBootApplication
class TechNewsApplication

fun main(args: Array<String>) {
    runApplication<TechNewsApplication>(*args)
}
